packages/fxa-auth-server/lib/senders/emails/global.scss

```diff
@@ -84,16 +88,21 @@ tr > td:first-child {
   font-family: $font-sans;
 }
 
-.header-text div {
+.text-xlarge div {
   @extend .text-xl;
   @extend .font-sans;
   text-align: center !important;
-  @extend .mb-5;
 }
 
-.primary-text div {
+.text-small div {
   @extend .text-sm;
-  @extend .mb-5;
   @extend .font-sans;
   text-align: center !important;
 }
+
+.text-xsmall div {
```

Comment:
We can probably call this one `text-footer`. Since the color is grey, I don't think we can reuse this elsewhere.